packages/components/src/panel/actions.native.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import {
+	TextControl,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import styles from './actions.scss';
+
+function PanelActions( { actions } ) {
+	return (
+		<View style={ styles.panelActionsContainer }>
+			{ actions.map( ( { label, onPress, labelStyle } ) => {
+				return (
+					<TextControl
+						label={ sprintf( __( '%s' ), label ) }
```

Comment:
That's not going to work for translations, as it would present the string `%s` to translators, not `label`. As @pinarol [mentioned](https://github.com/WordPress/gutenberg/pull/17703/files#r334925310), this needs to happen wherever the string is defined.

Reply:
Didn't know about it, thanks!